rc/extra/gpg.kak

```diff
@@ -0,0 +1,52 @@
+# Enable kakoune to open, edit, write files encrypted
+# using [GNU privacy guard](https://www.gnupg.org/)
+
+# options {
+declare-option str gpg_recipient 
+declare-option bool gpg_sign false
+# }
+
+hook -group gpg global BufOpenFile .*\.gpg %{
+    set buffer autoreload no
+    %sh{
+        d="$(dirname ${kak_buffile})"
+        f="$(basename -s .gpg ${kak_buffile})"
+        f="${d}/${f}"
+
+        umask 066
+        gpg --output "$f" --yes --decrypt ${kak_buffile}  2>/dev/null
+        printf %s\\n "
+            # Show in the modeline that this is a gpg file
+            evaluate-commands %{
+                e "$f"
+                hook -group gpg buffer BufWritePost %val{buffile} gpg-encrypt
+                hook -group gpg buffer BufClose %val{buffile} gpg-cleanup
+                echo  -markup {Information} Decrypted: "${kak_buffile}"
+            }
+        "
+    }
+}
+
+define-command -hidden gpg-encrypt %{
+    %sh{
+        options=""
+        [ -n "${gpg_recipient}" ] && options="-r ${gpg_recipient}"
+        [ "${gpg_sign}" = "true" ] && options="--sign ${options}"
+
+        cp -f "${kak_buffile}.gpg"{,~} 
```

Comment:
I'm not sure whether this shortcut is POSIX, is it? If not, prefer the long form.